.gitignore

```diff
@@ -13,6 +13,11 @@
 /**/.DS_store
 /cli/bin
 
+.yarn
+/.yarn
+*/.yarn
+**/.yarn
```

Comment:
**/.yarn/
should be enough to gitignore all .yarn files, pls remove other entries